rero_ils/modules/libraries/api.py

```diff
@@ -167,15 +167,18 @@ def _is_betweentimes(self, time_to_test, times):
                                             (time_to_test <= end_time))
         return times_open
 
-    def _has_is_open(self):
-        """Test if library has opening days."""
+    def _has_open_day(self):
+        """Test if library has opening days with standard times."""
         if opening_hours := self.get('opening_hours'):
             for opening_hour in opening_hours:
                 if opening_hour['is_open']:
                     return True
         if exception_dates := self.get('exception_dates'):
             for exception_date in exception_dates:
-                if exception_date['is_open']:
+                # ignore exception open dates if they have non-standard hours
+                # a non-standard open day is not considered open for circ
+                if exception_date['is_open'] \
+                        and not exception_date.get('times'):
```

Comment:
See next comment. Could be not relevant

Reply:
Thanks, I added an extended_validation method.